libraries/stdlib/src/kotlin/contracts/ContractBuilder.kt

```diff
@@ -146,4 +146,4 @@ public enum class InvocationKind {
 @InlineOnly
 @SinceKotlin("1.3")
 @Suppress("UNUSED_PARAMETER")
-public inline fun contract(builder: ContractBuilder.() -> Unit) { }
```

Comment:
Please revert unrelated line ending change

Reply:
OK done

Reply:
Hmm, strange, the new line is still there.
Ok, I'll merge the commit manually and clear it up.